### PR TITLE
First step - waypoint updater + DBW

### PR DESCRIPTION
**Pull request by bhiriyur**
Have a basic version of the following nodes:
- **Waypoint updater** (constant speed, not accounting for traffic lights/obstacles)
- **DBW controller** (not using PID but just adjusting throttle/brake when turning).

### CODE REVIEW

File: ros/src/waypoint_updater/waypoint_updater.py [hunk 1]
```diff
@@ -21,33 +21,114 @@
...
+                yj = wp.pose.pose.position.y
+                zj = wp.pose.pose.position.z
+
+                dij = math.sqrt( (xi-xj)**2 + (yi-yj)**2 + (zi-zj)**2 )
+
+                if dij < max_dist:
+                    max_dist = dij
+                    next_wp = i
+
+                wp_dist.append((dij, wp, count))
```
Comment (fspirit) [inline ros/src/waypoint_updater/waypoint_updater.py:90]: @bhiriyur  Why do we save count here? Is that for debugging of some kind?

File: ros/src/waypoint_updater/waypoint_updater.py [hunk 2]
```diff
@@ -21,33 +21,114 @@
...
+        msg = Lane()
+        msg.header.frame_id = '/world'
+        msg.header.stamp = rospy.Time.now()
+
+        max_vel = 40.0
+
+        for i in range(LOOKAHEAD_WPS):
+            wpi = self.wp_dist[i][1]
+            if i == 0:
+                rospy.loginfo("LOOKAHEAD: {} {} {}".format(i,
```
Comment (fspirit) [inline ros/src/waypoint_updater/waypoint_updater.py:119]: Are we trying to print out the closest waypoint for debugging?

File: ros/src/waypoint_updater/waypoint_updater.py (hunk 1 above)
Comment (bhiriyur) [reply ros/src/waypoint_updater/waypoint_updater.py:90]: Yes. It was for debugging. I had some loginfo there to see which waypoint was ahead and wanted to make sure we are publishing the final waypoints in the correct order after sorting.

File: ros/src/waypoint_updater/waypoint_updater.py (hunk 2 above)
Comment (bhiriyur) [reply ros/src/waypoint_updater/waypoint_updater.py:119]: Yes. Exactly.

**Pull request merged**